### PR TITLE
Keywords list added

### PR DESCRIPTION
**Pull request by cortezroberto**
## Description

- I've implemented the keywords list by creating a new file (_keywords.js_) inside the **./data** folder to manage all the keywords that the project is going to have.
- I've passed the simple variable to the JS variable to allow the jQuery Selector: **contains()** to read the keyword just as if was a String.
- I've implemented the keyword list to both files _newsAPIController.js_  and _twitterAPIController.js_ 

## Notes
I've not implemented the keywords list on line 67 of _newsAPIController.js_ because in that part of the module the function is only retrieving just one of the news lists so at this time is not possible to select a single keyword to be searched.
If I implement that then I would get one for each keyword when executing the **forEach** function to traverse the array, the main is just to get one.

In my opinion, it would be better if the project gets more issues to achieve this feature and also refactor all the functions.
I'd really appreciate a review/comment. Thanks in advance

### CODE REVIEW

**Review by MizouziE — CHANGES_REQUESTED** [review]
Hi @cortezroberto , thank you for sorting this out. I have a few comments on the following:
- `.vscode/launch.json` : Please remove this from the PR or better yet add it to `.gitignore`
- `controller/api/newsAPIController.js` : I see what you mean about line 67, but I believe you would actually be perfectly fine implementing the same here as you did above once you see my notes on changing the keyword array itself. The purpose of this endpoint is to gather **all** articles from a single `source` i.e. "The Guardian Newspaper" not just a single `article`.
- `data/keywords.js` : Here I think is where we can really *focus* in on the type of articles we want back. The ambiguity of the single word "energy" was leading to us getting nonsense articles included about stuff like "celebrity 'X' feels like they have more energy...blah blah" and it would be better to avoid that by being more specific and looking for "energy" **paired** with another word (i.e. 'energy price', 'energy prices', 'green energy', 'renewable energy', 'energy firm', 'energy company', 'energy tax'.... and so on) 
- I noticed that a formatter has been used, which is fine because we have another job being done to create a prettier config file and I think we'll end up matching the format you've used anyway 😅
- Finally, I'm afraid there was already a fairly major refactor done as I believe you suggested and it has lead to a few merge conflicts. Would you be able to rebase and ensure that all the changes you've made remain because they look good to me!

Thanks again for this and my apologies for the need to rebase 🙏

**Comment by cortezroberto** [comment]
Got it @MizouziE , I'll take a look to the recommendations and work on the changes!
Please let me know if there's more comments.👍

Thank you 😁
Greetings.

**Comment by MizouziE** [comment]
@cortezroberto Thanks man!

**Comment by MizouziE** [comment]
@cortezroberto Most certainly. I will have a look ASAP and get back to you.

**Comment by MizouziE** [comment]
@cortezroberto So I've had a look and this is what I think you'll need to do...

The error messages I'm getting in the terminal are:
image

What that is relating to is these files highlighted in red:
image

As you can see, they were *moved* so git see's it as deleted. They already exist in their proper places, so you can delete the old (red) ones. The `keywords.js` looks like it will automatically sort itself out, if I am reading the error message correctly.

The only other thing I noticed is that your `.vscode` directory is showing up so change line 4 in the `.gitignore` to just `.vscode` and that should fix that.

After making those changes do another commit and push and it should be good to go.

🤞

**Comment by MizouziE** [comment]
Hi @cortezroberto , it's still having conflicts because there are a number of commits missing from your repo. What I would try is:
1. Fetch upstream on **your** master branch to bring it in line with this one we want to merge into. 
2. Rebase your keyboard-check branch on your master branch

You may have a few conflicts to sort out, but you'll be able to see where the changes you made need to be, just delete the lines that your changes are missing from and git add <fixed filename> then git rebase --continue

Once all conflicts are clear, push again to your remote keyboard-check branch and we _should_ be all good. 

Rebasing is painful, I'm sorry man. But **this** will make you an absolute expert! 😅

**Comment by cortezroberto** [comment]
Hi @MizouziE , I have no conflicts already, finally 😅.
I just need the workflow approval and I am open to getting feedback or any changes 😁

Thank you!
Greetings.

File: src/data/keywords.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+//List of keywords
+module.exports = keywords = [
+  "green energy",
+  "energy price",
+  "energy firms",
+  "renewable energy",
+  "energy company",
+  "energy tax",
```
Comment (MizouziE) [inline src/data/keywords.js:8]: The layout is also great because it makes it easier to add to later 🤙

**Review by MizouziE — APPROVED** [review]
Great stuff, and most importantly...

🎉🎉🎉🎉🎉🎉🎉🎉
NO MORE MERGE CONFLICTS
🥳🥳🥳🥳🥳🥳🥳🥳

I'm sorry it ended up being so difficult, but this is gonna be such a huge improvement it was worth it! 💪

**Comment by cortezroberto** [comment]
Thank you @MizouziE . I learned a lot about Merge conflicts! 😁
I will look the issues to pick a new one.

Thanks again.
Greetings!

**Comment by MizouziE** [comment]
@cortezroberto It has been a pleasure! I've learned A LOT too!

**Pull request merged**